rfcs/0008-node-discovery/0008-node-discovery.md

```diff
@@ -0,0 +1,92 @@
+---
+Number: 8
+Category: Standards Track
+Status: Draft
+Author: Linfeng Qian / JinYang Jiang
+Organization: Nervos Foundation
+Created: 2018-11-28
+---
+
+# CKB Node Discovery Protocol
+
+CKB node discovery protocol mainly refers to [Satoshi Client Node Discovery][0]. The differences between them are summarized below:
+
+* The node version number is included in the `GetNodes` message.
+* The `Nodes` message is used to periodically broadcast all nodes currently connected.
+* We use `multiaddr` as the format of node addresses  (There is no `/p2p/` field here and if it's considered as *misbehavior*, a low score will be given.)
```

Comment:
```suggestion
* We use `multiaddr` as the format of node addresses  (It MUST NOT include `/p2p/` segment otherwise it's considered as *misbehavior* and a low score will be given.)
```